web/app.js

```diff
@@ -1956,6 +1983,14 @@ function webViewerRotateCw() {
 function webViewerRotateCcw() {
   PDFViewerApplication.rotatePages(-90);
 }
+function webViewerSwitchScrollMode(evt) {
+  PDFViewerApplication.pdfViewer.setScrollMode(evt.mode);
+}
+function webViewerToggleSpreadMode(evt) {
+  const { pdfViewer, } = PDFViewerApplication;
+  pdfViewer.spreadMode = pdfViewer.spreadMode === evt.mode ? SpreadMode.OFF :
```

Comment:
I'm not a UX guy, so please take this with a grain of salt (and get opinions from other project members).
For UI consistency, wouldn't it be better to have an explicit menu item for `OFF`? Then you can also replace the `spreadMode` getter and setter with a `setSpreadMode` method, similarly implemented as `setScrollMode`.

Reply:
I like it the way it is; I think having ‘Lock Odd/Even Spreads’ as toggle buttons that can be clicked on or off is more intuitive and less cluttered. You always have a scrolling mode, just like you always have an active tool, so using the same radio-button-style UX for those modes makes sense, but I didn't think people would naturally feel like they always have a ‘spread mode’, even if that's how it's implemented technically. They can choose to lock spreads or not. It's a trivial change though, so if consensus is against me, I can add a sixth menu item for it.